src/nxdoc/web-ui/nuxeo-elements/quality-assurance.md

```diff
@@ -293,71 +297,20 @@ Scenario: Admin center
-
-It is a complete browser (End-to-End) testing solution which aims to simplify the process of setting up Continuous Integration and writing automated tests. Nightwatch can also be used for writing Node.js unit tests.
-
-#### Browser/IDE Setup
-
-Here is the [Nightwatch.js wiki](https://github.com/nightwatchjs/nightwatch/wiki) for more information about browser test setup and running/debugging in your favorite IDE.
-
-#### Configuration
-
-The test runner expects a configuration file to be passed, using by default a `nightwatch.json` file from the current directory, if present. A `nightwatch.conf.js` file will also be loaded by default, if found. Click [here](http://nightwatchjs.org/gettingstarted#settings-file) to view an example.
-
-#### Page Object API
-
-In order to factorize your tests, the Page Object API has been introduced in Nightwatchjs and allows developers to create different page descriptors with the related selectors for re-using them easily in the IT tests. Click [here](https://github.com/nightwatchjs/nightwatch/wiki/Page-Object-API) for more information.
-
-#### The Command Queue
-
-When Nightwatch runs a test, it processes its commands in a list known as the command queue. This list manages the asynchronous execution of the commands defined in that test. Click [here](https://github.com/nightwatchjs/nightwatch/wiki/Understanding-the-Command-Queue) for more information.
-
-#### API
-
-Please refer to this [developer guide](http://nightwatchjs.org/guide) to see how to use:
-
-- Default commands
-- CSS or XPath Selectors
-- Pages
-- APIs
-
-#### Nuxeo Maven Package Example
-
-[An example](https://github.com/nuxeo/nuxeo-marketplace-sample/edit/master/ftest/nightwatchjs) of a Nuxeo package to run with Maven and npm nightwatchjs tests.
-
-## Security Strategies
-
-### Search: GET method URL
-
-With Nuxeo platform, you have two ways to fetch data from the server for building listings and searching:
-
-- Via 'direct' queries
-- Via [`pageproviders`]({{page version='' space='nxdoc' page='page-providers'}})
-
-If your network environment (proxies, firewalls) doesn't allow queries inside URL for `GET` Method for security purpose, you have to avoid the `nuxeo-resource` usage with the endpoint `query` if you want to execute a search via 'direct' queries:
-
-See this GET Method URL example:
-
-```
-http://NUXEO_SERVER/nuxeo/api/v1/query?currentPageIndex=0&pageSize=10&query=SELECT%20%2A%20FROM%20Document%20where%20ecm%3AprimaryType%20in%20%28%27UserProfile%27%29%20%20
-```
-
-**We recommend that you**:
-
-- use page providers in general (see above in `Performance Strategies > Search`)
-- use [`Repository.Query`](http://explorer.nuxeo.com/nuxeo/site/distribution/latest/viewOperation/Repository.Query) operation with [`nuxeo-operation`](https://www.webcomponents.org/element/nuxeo/nuxeo-elements/elements/nuxeo-operation) element if you still want to search via 'direct' queries.
```

Comment:
Is there any good reason to remove this remark about security strategies?

Reply:
most certainly a mistake. I will revert it.